apps/style/applab/style.scss

```diff
@@ -55,12 +76,19 @@ div#visualization {
 }
 
 @media screen and (min-width: 1001px) and (max-width: 1050px) {
+  $width: 250px;
   div#visualization.responsive {
-    max-height: 250px;
-    max-width: 250px;
+    max-height: $width;
+    max-width: $width;
   }
   div#visualizationColumn.responsive {
-    max-width: 250px;
+    max-width: $width;
+  }
+  div#codeWorkspace {
+    left: $width;
+  }
+  html[dir='rtl'] div#codeWorkspace {
```

Comment:
optional CSS trick for next time: change this line to "html[dir='rtl'] & {" and move it into the div#codeWorkspace block above.